pkgs/development/interpreters/php/default.nix

```diff
@@ -235,6 +148,12 @@ let
       inherit sha256;
     };
 
+    patches = [ ./fix-paths-php7.patch ] ++ extraPatches;
+
+    stripDebugList = "bin sbin lib modules";
```

Comment:
Any reason why we don't move debugging-symbols into a separate `debug`-output (is usually doable with `separateDebugInfo = true`)?

Reply:
This isn't actually any change except that I moved it around in the file to make it look better (in my opinion).

May be doable, haven't tried.

Reply:
Seems to work fine, fixed in 0d392490483